src/compositemark/callout.ts

```diff
@@ -0,0 +1,131 @@
+import {Config} from '../config';
+import {Encoding} from '../encoding';
+import * as log from '../log';
+import {MarkConfig, MarkDef} from '../mark';
+import {GenericUnitSpec, LayerSpec} from '../spec';
+
+
+export const CALLOUT: 'callout' = 'callout';
+export type CALLOUT = typeof CALLOUT;
+
+export interface CalloutDef {
+  type: CALLOUT;
+  calloutAngle?: number;
```

Comment:
You should share things that are similar between `CalloutDef` and `CalloutConfig` so that you don't have the write documentation twice.  

Also, missing documentation for all of these properties here.